moped-toolbox/geometry_prototype/create_prototype.sql

```diff
@@ -0,0 +1,141 @@
+create table projects (
+  id serial primary key,
+  name character varying
+  );
+  
+create table layers (   
+    id serial primary key,
+    internal_table character varying, -- I want to call this "table" but that's a very reserved word. Any other ideas? 
+    reference_layer_primary_key_column character varying
```

Comment:
i think `name` or `layer_name` or `reference_table_name` are good.

also think it's safe to use `pk` for that second column: `reference_layer_pk_column`